.devcontainer/devcontainer.json

```
@@ -3,7 +3,7 @@
 {
 	"name": "Node.js & TypeScript",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/typescript-node:0-16",
+	"image": "mcr.microsoft.com/devcontainers/typescript-node:0-20",
```

```suggestion
	"image": "mcr.microsoft.com/devcontainers/typescript-node:1-20-bookworm",
```

Looks like this one is using an older image version. Can you help update this as well? Thanks!